docs/api/example-0.0.2.tar.gz-preview.txt

```diff
@@ -17,6 +17,7 @@
 91348 example-0.0.2/kibana/index-pattern/filebeat.json
 0 example-0.0.2/kibana/infrastructure-ui-source/
 797 example-0.0.2/kibana/infrastructure-ui-source/default.json
+0 example-0.0.2/kibana/search/
```

Comment:
@mtojek Kind of suprised about this change. Lets see what CI thinks.

Reply:
Turns out I had locally still an empty directory probably which is not checked into git. Cleaning up the testdata directory and checking it out again solved this issue (I hope).